src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using InstagramClone.Api.Database;
+using InstagramClone.Api.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class UsersController : ControllerBase
+    {
+        private readonly InstagramCloneDbContext _context;
+        public UsersController(InstagramCloneDbContext context)
+        {
+            _context = context;
+        }
+        
+        [HttpGet("{Id}")]
```

Comment:
you need to check if this is case-sensitive. 

is `"{Id}"` the same as `"{id}"`?

Reply:
I have checked and made the right fix.